javascript/statements.json

```
@@ -1864,10 +1864,12 @@
                 }
               ],
               "firefox": {
-                "version_added": false
+                "version_added": false,
+                "notes": "See <a href='https://bugzil.la/1247687'>bug 1247687</a>"
```

I'm sorry that this is the smallest nit pick possible, but I can't help myself. Once this is done, we can merge. 😄

```suggestion
                "notes": "See <a href='https://bugzil.la/1247687'>bug 1247687</a>."
```

---

Fixed.